vispy/visuals/_scalable_textures.py

```diff
@@ -273,12 +290,11 @@ def scale_and_set_data(self, data, offset=None, copy=True):
         is_auto = isinstance(clim, str) and clim == 'auto'
         if data.ndim == self._ndim or data.shape[self._ndim] == 1:
             if is_auto:
-                clim = np.min(data), np.max(data)
-            clim = (np.float32(clim[0]), np.float32(clim[1]))
```

Comment:
Why isn't this float conversion needed anymore?

Reply:
I'm not sure why it was ever this way. It seems weird to use numpy types in normal Python properties, so I figured I'd try removing it.

Reply:
I think it is less weird considering it is formatting them to be used on the GPU. At least that was my assumption when I saw it. I think something on the lower level gloo layers was complaining about ints being passed or 64-bit floats or something.

Reply:
Yeah, but then this should be handled there, I'd think. I now updated to at least always be float (no ints). If you prefer putting back the `np.float32()` then I could do that though.

Reply:
If there are no warnings or errors from doing `float(x)` then I'm OK with this.